TestMissingAnyFills::TestMonth random failures

### Description
<!--Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.-->

TestMissingAnyFills::test_month tends to fail intermittently, particularly on the macOS Python3.7 build but also on Linux and Windows at a lesser frequency. Caution suggests that if this specific case is failing sometimes, there must either be a bug within xclim or xarray or there is some RNG-dependent call or test setup that we've coded that has some failing edge cases.

I'm not sure what the urgency of this bug is but it's been plaguing TravisCI builds for several months so I'm opening an issue to at least have this on the board. Here's the stack:
```
________________________ TestMissingAnyFills.test_month ________________________
self = <test_checks.TestMissingAnyFills object at 0x11b253150>
tasmin_series = <function tasmin_series.<locals>._tasmin_series at 0x11a409f80>

    def test_month(self, tasmin_series):
        ts = tasmin_series(np.zeros(36))
        miss = checks.missing_any(ts, freq="YS", month=7)
        np.testing.assert_equal(miss, [False])
    
        miss = checks.missing_any(ts, freq="YS", month=8)
        np.testing.assert_equal(miss, [True])
    
        with pytest.raises(ValueError, match=r"No data for selected period."):
            miss = checks.missing_any(ts, freq="YS", month=1)
    
        miss = checks.missing_any(ts, freq="YS", month=[7, 8])
        np.testing.assert_equal(miss, [True])

        ts = tasmin_series(np.zeros(76))
        miss = checks.missing_any(ts, freq="YS", month=[7, 8])
>       np.testing.assert_equal(miss, [False])

E       AssertionError: 
E       Items are not equal:
E        ACTUAL: <xarray.DataArray (time: 1)>
E       array([ True])
E       Coordinates:
E         * time     (time) datetime64[ns] 2000-01-01
E        DESIRED: [False]
```

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -9,6 +9,7 @@ History
 * Reverted #311, removing the `_rolling` util function. Added optimal keywords to `rolling()` calls.
 * Fixed `ensembles.create_ensemble` errors for builds against xarray master branch.
 * Reformatted code to make better use of Python3.6 conventions (f-strings and object signatures).
+* Fixed randomly failing tests of `checks.missing_any`.
 
 0.12.x-beta (2019-11-18)
 ------------------------

--- a/xclim/checks.py
+++ b/xclim/checks.py
@@ -201,7 +201,7 @@ def missing_any(da, freq, **indexer):
         else:
             t = pd.date_range(t0, t1, freq="D")
 
-        sda = xr.DataArray(data=np.empty(len(t)), coords={"time": t}, dims=("time",))
+        sda = xr.DataArray(data=np.ones(len(t)), coords={"time": t}, dims=("time",))
         st = generic.select_time(sda, **indexer)
         sn = st.notnull().resample(time=freq).sum(dim="time")
         miss = sn != c

